package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redo-monolith",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
I don't know. I think it's a GH issue because the same commit exists in this branch and in `master` :man_shrugging: